versioned_docs/version-8.7/self-managed/identity/application-user-group-role-management/manage-groups.md

```diff
@@ -1,21 +1,34 @@
 ---
-id: manage-users-roles-to-group
-title: "Manage users and roles in a group"
-description: "Groups are a way to apply a set of roles and authorizations to users. Use Identity to manage users and roles in a group."
+id: manage-groups
+title: "Manage groups"
+description: "Groups are a way to apply a set of roles and permissions to users. Use Identity to create a group."
 ---
 
-[Users](/self-managed/identity/user-guide/roles/manage-roles.md) and [roles](/self-managed/identity/user-guide/roles/manage-roles.md) can be added or removed from groups.
+Groups are a way to group a set of members. A set of [roles](/self-managed/identity/application-user-group-role-management/manage-roles.md) can be mapped to groups.
```

Comment:
This here is our statement how users, groups, and roles relate and why I should care as an administrator. I think we can make this more clear. I don't have a good suggestion right now, but maybe with a fresh view next week.

Reply:
See what you mean. Proposing the following:
> Groups allow you to efficiently manage access by organizing users (group members) and assigning [roles](/self-managed/identity/application-user-group-role-management/manage-roles.md). Instead of assigning roles to individual users, you can map a set of roles to a group, and all group members will automatically inherit those permissions.

Reply:
Sounds good!